{"version":"3.8.0","messages":["\nAn error occurred while loading ./spec/pry_debugging_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0m\u001b[32mdef\u001b[0m \u001b[1;34mplus_two\u001b[0m(num)\u001b[0m\n\u001b[31m  \t\u001b[0mnum + \u001b[1;34m2\u001b[0m\u001b[0m\n\u001b[31m  \t\u001b[0mnum\u001b[0m\n\u001b[31m  \t\u001b[0mbinding.pry\u001b[0m\n\u001b[31m  \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mLoadError:\u001b[0m\n\u001b[31m  cannot load such file -- \u001b[0G[1] pry(main)> exi\u001b[0m\n\u001b[31m  [1] pry(main)> rError: Input/output error - read\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `block in input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:115:in `interruptible_region'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:198:in `input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:184:in `block in read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:129:in `handle_read_errors'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:170:in `read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:98:in `read'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:68:in `block in repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `loop'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `block in start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:61:in `__with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:79:in `with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:13:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/pry_class.rb:192:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/cli.rb:116:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/bin/pry:12:in `<top (required)>'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `load'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `<main>'\u001b[0m\n\u001b[31m  [1] pry(main)> Error: Input/output error - read\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `block in input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:115:in `interruptible_region'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:198:in `input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:184:in `block in read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:129:in `handle_read_errors'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:170:in `read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:98:in `read'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:68:in `block in repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `loop'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `block in start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:61:in `__with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:79:in `with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:13:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/pry_class.rb:192:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/cli.rb:116:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/bin/pry:12:in `<top (required)>'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `load'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `<main>'\u001b[0m\n\u001b[31m  [1] pry(main)> Error: Input/output error - read\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `block in input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:115:in `interruptible_region'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:198:in `input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:184:in `block in read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:129:in `handle_read_errors'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:170:in `read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:98:in `read'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:68:in `block in repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `loop'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `block in start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:61:in `__with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:79:in `with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:13:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/pry_class.rb:192:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/cli.rb:116:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/bin/pry:12:in `<top (required)>'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `load'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `<main>'\u001b[0m\n\u001b[31m  [1] pry(main)> Error: Input/output error - read\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `block in input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:115:in `interruptible_region'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:198:in `input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:184:in `block in read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:129:in `handle_read_errors'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:170:in `read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:98:in `read'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:68:in `block in repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `loop'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `block in start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:61:in `__with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:79:in `with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:13:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/pry_class.rb:192:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/cli.rb:116:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/bin/pry:12:in `<top (required)>'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `load'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `<main>'\u001b[0m\n\u001b[31m  [1] pry(main)> Error: Input/output error - read\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:199:in `block in input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:115:in `interruptible_region'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:198:in `input_readline'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:184:in `block in read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:129:in `handle_read_errors'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:170:in `read_line'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:98:in `read'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:68:in `block in repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `loop'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:67:in `repl'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `block in start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:61:in `__with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/input_lock.rb:79:in `with_ownership'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:38:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/repl.rb:13:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/pry_class.rb:192:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/lib/pry/cli.rb:116:in `start'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/gems/pry-0.11.3/bin/pry:12:in `<top (required)>'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `load'\u001b[0m\n\u001b[31m  /usr/local/rvm/gems/ruby-2.3.1/bin/pry:23:in `<main>'\u001b[0m\n\u001b[31m  FATAL: Pry failed to get user input using `Readline`.\u001b[0m\n\u001b[31m  To fix this you may be able to pass input and output file descriptors to pry directly. e.g.\u001b[0m\n\u001b[31m    Pry.config.input = STDIN\u001b[0m\n\u001b[31m    Pry.config.output = STDOUT\u001b[0m\n\u001b[31m    binding.pry\u001b[0m\n\u001b[36m# ./lib/pry_debugging.rb:1:in `<top (required)>'\u001b[0m\n\u001b[36m# ./spec/pry_debugging_spec.rb:2:in `<top (required)>'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.00031958,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":1},"summary_line":"0 examples, 0 failures, 1 error occurred outside of examples"}